index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="./normalize.css"/>
+    <link rel="stylesheet" type="text/css" href="./style.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
+    <script src="./script.js" defer></script> 
+    <title>Surfing Penguins - JS Game. </title>
+</head>
+<body>
+    <main>
+        <div class="landingTopContainer">
+            <div class="landingLeft"></div>
+            <div class="landingTitleContainer">
+                <h1 class="landingTitle">SURFWAVE Fury</h1>
+            </div>
+            <div class="leaderSurfboardContainer">
+                <button id="leaderSurfboard">Leader Surf Board</button>
+            </div>
```

Comment:
This shouldn't be here yet! Not a part of this story

Reply:
Commented out as discussed !